front/app/components/EventCard/EventInformation/index.tsx

```diff
@@ -140,7 +158,9 @@ const EventInformation = ({ event, goToEvent, titleFontSize }: Props) => {
             </Box>
           )}
           {onlineLink && (
-            <Box display="flex" mb="12px">
+            // The zIndex and position relative are needed to make sure
+            // the link is clickable by moving it to the top of the card
```

Comment:
I have the context of this PR so I get why you are doing this, but somebody who is new to this code won't get the why. Let's add it here.